protos/route.proto

```diff
@@ -145,6 +145,7 @@ message CreatePeerResponse {
 message MirrorStatusRequest {
   string flow_job_name = 1;
   bool include_flow_info = 2;
+  bool exclude_batches = 3;
```

Comment:
nit: `exclude_cdc_batches`

Reply:
this'll also be excluding qrep batches in near future